### ISSUE TITLE
[Screen Reader-Help-Projects] 'Alt' attribute is not defined for the images like Flashing Heart, Smiley buttons on Projects screen.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2588]
**User Experience:**
User with Visual disabilities might not able to understand the image if 'Alt' Text is not defined. Like in the scenario alt text is not defined for the images.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Screen reader:** Narrator

**Repro Steps**
1-Navigate to url:- https://makecode.microbit.org/acc
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'Projects' control and select it
5-Verify alt attribute is defined for the images by using developer tool.

**Actual Result**
'Alt' attribute is not defined for the images like Flashing Heart, Smiley buttons on Projects screen.

**Expected Result**
'Alt' attribute should be defined for the images like Flashing Heart, Smiley buttons on Projects screen.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%201.1.1%20%E2%80%93%20Non-text%20Content.docx?d=wd2d2051fbdc84af78e1838aae867e216&csf=1

**Suggested fix:**
In HTML developers can ,combine adjacent image and text links for the same resource, Use alt attributes on img elements , Use the body of the object element. 
e.g. 
<  img src="newsletter.gif" alt="Free newsletter. Get free recipes, news, and more. Learn more." /  >
 
<  object classid="http://www.example.com/animatedlogo.py"
  <  img src="staticlogo.gif" alt="Company Name" / >
/  object>
**Key References for Fix:** 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html 

**Refer Attachment**
![mas20_projects_alt is not defined](https://user-images.githubusercontent.com/30459830/28774915-250035ca-760d-11e7-961b-7aae1d5f523d.png)

**Comment by abchatra** [comment]
@v-mesala once you add label remove the reference from the title. For example remove [AllyBlocking]  and others from the title in this issue

**Comment by v-amik** [comment]
Review done

**Comment by v-mesala** [comment]
Bug is verified on URL :https://makecode.microbit.org/acc bug is fixed hence closed the bug and added the closed tag.
